pkg/centos8/Dockerfile

```diff
@@ -0,0 +1,218 @@
+ENV LIBCURL_TEMP=/tmp/libcurl
+ENV PATH=/opt/hubble/bin/:/opt/hubble/include:/opt/hubble/lib:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+RUN mkdir -p "$LIBCURL_TEMP" \
+ && cd "$LIBCURL_TEMP" \
+ && git clone "$LIBCURL_SRC_URL" \
+ && cd curl \
+ && git checkout "$LIBCURL_SRC_VERSION" \
+ && ./buildconf \
+ && ./configure --prefix=/opt/hubble --disable-ldap --without-nss --disable-manual --disable-gopher --disable-smtp --disable-smb --disable-imap --disable-pop3 --disable-tftp --disable-telnet --disable-dict --disable-ldaps --disable-ldap --disable-rtsp --with-libssh \
+ && make \
+ && make install
+
+#git install start
+#install git so that git package won't be a package dependency
+#requires make git libcurl-devel autoconf zlib-devel gcc
+ENV GIT_SRC_URL=https://github.com/git/git.git
+ENV GIT_SRC_VERSION=v2.21.0
+ENV GITTEMP=/tmp/gittemp
+RUN mkdir -p "$GITTEMP" \
+ && cd "$GITTEMP" \
+ && git clone "$GIT_SRC_URL" \
+ && cd git \
+ && git checkout "$GIT_SRC_VERSION" \
+ && make configure \
+ && ./configure --prefix=/opt/hubble --with-tcltk=no --with-expat=no --with-python=no  --with-curl=/opt/hubble \
+ && echo "NO_TCLTK=YesPlease" >> config.mak.autogen \
+ && echo "NO_PERL=YesPlease" >> config.mak.autogen \
+ && sed -i '0,/^NO_GETTEXT/s/^NO_GETTEXT.*/NO_GETTEXT=YesPlease/' config.mak.autogen \
+ && make \
+ && make install
+
+#clean up of /opt/hubble
+RUN rm /opt/hubble/bin/curl* \
+ && rm -rf /opt/hubble/include /opt/hubble/share
+
+#libgit2 install start
+#must precede pyinstaller requirements
+ENV LIBGIT2_SRC_URL=https://github.com/libgit2/libgit2/archive/v0.26.5.tar.gz
+#it turns out github provided release files can change. so even though the code hopefully hasn't changed, the hash has.
+ENV LIBGIT2_SRC_SHA256=52e28a5166564bc4365a2e4112f5e5c6e334708dbf13596241b2fd34efc1b0a9
+ENV LIBGIT2_SRC_VERSION=0.26.5
+ENV LIBGIT2TEMP=/tmp/libgit2temp
+RUN mkdir -p "$LIBGIT2TEMP" \
+ && cd "$LIBGIT2TEMP" \
+ && wget -q "$LIBGIT2_SRC_URL" -O libgit2.tar.gz \
+ && echo "$LIBGIT2_SRC_SHA256  libgit2.tar.gz" | sha256sum -c - \
+ && tar xzf libgit2.tar.gz \
+ && cd libgit2-"$LIBGIT2_SRC_VERSION"/ \
+ && cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/ \
+ && make \
+ && make install
+
+#pyinstaller requirements start
+#must be preceded by libgit2 install
+COPY pyinstaller-requirements.txt /
+#default python-pip from yum does not like upgrading itself from pip. looking for better options other than wget.
+RUN wget -c https://bootstrap.pypa.io/get-pip.py \
+ && python get-pip.py \
+ && pip -v install -r pyinstaller-requirements.txt
+
+#fpm package making requirements start
+RUN dnf install -y ruby ruby-devel rpm-build rubygems gcc make \
+ && gem install --no-ri --no-rdoc fpm
+
+#pyinstaller start
+#commands specified for ENTRYPOINT and CMD are executed when the container is run, not when the image is built
+#use the following variables to choose the version of hubble
+ENV HUBBLE_CHECKOUT=v3.0.11
+ENV HUBBLE_VERSION=3.0.11
```

Comment:
please confirm if it will be 3.0.11 or 3.0.12

Reply:
Done, please check